CodingStyle

```diff
@@ -28,12 +28,12 @@ by section.
 
     Google uses CamelCaps for all type names.  We use two naming schemes:
 
-    - for naked structs (simple data containers), lower case with _d
-      suffix ('d' for data).  Not _t, because that means typdef.
+    - for naked structs (simple data containers), lower case with _t.
+      Yes, _t also means typedef.  It's perhaps not ideal.
```

Comment:
How do you feel about adding "or none at all" (as is done below for data members)? There's ample precedent "in the wild" and there's not really syntactic ambiguity.

Reply:
you mean no suffix at all? sure.  i don't think there is much of this, though.

Reply:
Right, that's what I mean. And I don't think there's a lot of that in Ceph's code directly.